hack/build-go.sh

```diff
@@ -23,6 +23,8 @@ source hack/common.sh
 source hack/config.sh
 source hack/version.sh
 
+source hack/go-build-functests.sh
```

Comment:
Is sourcing here a left-over or do you want to build in addition to all go binaries too?

Reply:
This is to keep current behavior.

See: https://github.com/kubevirt/kubevirt/blob/8b7377e297436489f021c021e473d03997f5d5d9/hack/build-go.sh#L147

Reply:
I see. So you want to have a break-out target but still build it together with the rest as well.

Reply:
Exactly!